### ISSUE TITLE
[Bug][About-Foundation-Admin] Missing "Alt text" field for image in "Liatoshynski Foundation" section on "About the Foundation" page

### ISSUE THREAD

**Issue by nadiia-dotsiak** [Liatoshynsky-Foundation/lf-manual-tests#530]
**Summary:** Missing "Alt text" field for image in "Liatoshynski Foundation" section on "About the Foundation" page

**Environment:** Stage

**Reproducible:** always

**Build found:** last commit from 

**Preconditions:**
 Admin is logged in.

**Steps**
1. Go to [About Us](https://lf-admin-stage-dzfpbxeuauh7fad2.polandcentral-01.azurewebsites.net/about-us) page.
2. Open the **Liatoshynski Foundation** section for editing content.
3. Locate the image edit/change area.

**Actual result:**

The Admin panel does not provide an Alt text field for the image in the Liatoshynski Foundation section.

<img width="452" height="157" alt="Image" src="https://github.com/user-attachments/assets/685d7295-d888-4a03-b0de-126f864f6df7" />

**Expected result:**

The Admin panel should provide an Alt text field for the image, allowing administrators to enter descriptive alternative text in accordance with the design and accessibility requirements.

<img width="441" height="180" alt="Image" src="https://github.com/user-attachments/assets/9c08ba53-0eb9-4857-9724-572935fe64c4" />

**User story and test case links**
User story [#1426](https://github.com/Liatoshynsky-Foundation/lf-client/issues/1426#issue-4702521014), [#1428](https://github.com/Liatoshynsky-Foundation/lf-client/issues/1428)
TC [#430](https://github.com/Liatoshynsky-Foundation/lf-manual-tests/issues/430)
